waku/common/wakubridge.nim

```diff
@@ -22,7 +23,24 @@ import
   # Common cli config
   ./config_bridge
 
-const clientIdV1 = "nim-waku v1 node"
+const
+  clientIdV1 = "nim-waku v1 node"
+  defaultBridgeTopic = "/waku/2/default-bridge/proto"
+  defaultTTL = 5'u32
+
+proc toWakuMessage(env: Envelope): WakuMessage =
+  # Translate a Waku v1 envelope to a Waku v2 message
+  WakuMessage(payload: env.data,
+              contentTopic: ContentTopic(uint32.fromBytes(env.topic)),
+              version: 1)
+
+proc toWakuV2(env: Envelope, nodev2: WakuNode) {.async.} =
+  await nodev2.publish(defaultBridgeTopic, env.toWakuMessage())
```

Comment:
Wondering why not using `defaultTopic` https://github.com/status-im/nim-waku/blob/0050f9da59a129bfab34c5eaaa03f864afd01034/waku/v2/node/wakunode2.nim#L35 of wakunode2 in here?

Reply:
For now I'd like to keep the default topic for bridging messages separate from the one we use for everything else. This may certainly change once I implement a standalone WakuBridge.